core/plugins/org.fusesource.ide.camel.model.service.core/src/org/fusesource/ide/camel/model/service/core/util/CamelCatalogUtils.java

```diff
@@ -126,6 +129,26 @@
 			PURE_FIS_CAMEL_VERSIONS.put("2.18.1.redhat-000012", "2.2.170.redhat-000010");			
 			PURE_FIS_CAMEL_VERSIONS.put(DEFAULT_CAMEL_VERSION,  "2.2.170.redhat-000013");
 		}
+
+		try {
+			Properties versionMapping = new Properties();
+			URL url = new URL(TEST_VERSIONS_URL);
+			versionMapping.load(url.openStream());
+		
+			for(String camelVersion : versionMapping.stringPropertyNames()) {
+				TEST_CAMEL_VERSIONS.add(camelVersion);
+			}
+		} catch (IOException ex) {
+			CamelModelServiceCoreActivator.pluginLog().logError("Unable to retrieve the Test Camel Versions list from online repo. Falling back to defaults.", ex);
+
+			TEST_CAMEL_VERSIONS.add("2.17.0.redhat-630187");
+			TEST_CAMEL_VERSIONS.add("2.17.0.redhat-630224");
+			TEST_CAMEL_VERSIONS.add("2.17.7");
+			TEST_CAMEL_VERSIONS.add("2.18.1.redhat-000012");
+			TEST_CAMEL_VERSIONS.add(DEFAULT_CAMEL_VERSION);
```

Comment:
i would prefer to use the "2.18.1.redhat-000015" and having then a check if the DEFAULT_CAMEL_VERSION is part of the TEST_CAMEL_VERSIONS